README.md

```diff
@@ -49,6 +50,49 @@ The plugin requires an active Parse.ly account. Parse.ly gives creators, markete
 
 To run the plugin locally or to contribute to it, please check the instructions in the [CONTRIBUTING](CONTRIBUTING.md) file.
 
+## REST API
+
+The plugin adds a `parsely` field to certain REST API responses. This can be disabled by returning `false` from the `wp_parsely_enable_rest_api_support` filter.
+
+Example:
+
+```
+// Disable all REST API output from the Parse.ly plugin.
+add_filter( 'wp_parsely_enable_rest_api_support', '__return_false' );
+```
+
+The plugin adds the `parsely` field to endpoints corresponding to the Tracked Post Types and Tracked Page Types selected in the plugin settings. By default, this would be the `/wp-json/wp/v2/pages` and `/wp-json/wp/v2/posts` endpoints along with the corresponding single resource endpoints.
+
+This choice of objects types can be further changed by using the `wp_parsely_rest_object_types` filter.
+
+Example:
+
+```
+// Disable REST API output from pages, but enable for term archives.
+add_filter(
+	'wp_parsely_rest_object_types',
+	function( $object_types ) {
+		$object_types = array_diff( $object_types, array( 'page' ) );
+		$object_types[] = 'term';
+		return $object_types;
+	}
+);
+```
+
+The `parsely` field contains the following fields:
+ - `version`, which is a string identifying the version of the REST API output; this will be updated if changes are made to the output, so consuming applications can check against it.
+ - `meta`, which is an array of metadata for the specific post, page or other object type.
+ - `rendered`, which is the rendered HTML of the metadata for the post, page or other object type. This will be a JSON-LD `<script>` tag, or a set of `<meta>` tags, depending on the format selected in the plugin settings. The decoupled code can consume and use this directly, instead of building the values from the `meta` field values.
+
+The `rendered` field is a convenience field is only included if the `wp_parsely_enable_rest_rendered_support` filter returns `true`. To disable this, return `false` from the filter.
```

Comment:
```suggestion
The `rendered` field is a convenience field containing the HTML-formatted meta data which can be printed to a decoupled page as is.

This can be disabled by returning `false` from the `wp_parsely_enable_rest_rendered_support` filter.
```